common/scripted_triggers/wc_maa_triggers.txt

```diff
@@ -91,6 +91,12 @@ land_skirmish_culture_trigger = {
 	}
 }
 
+like_arachnathid_trigger = {
+	this = culture:nerubian
+	culture = {
+		has_innovation = innovation_arachnathid
+	}
```

Comment:
This may be a fix to errors I reported above.
```suggestio
	culture = {
		this = culture:nerubian
		has_innovation = innovation_arachnathid
	}
```
Other triggers should be fixed the same way.
`this` must be directed to `culture`. So `this` must be nested inside `culture`.

Reply:
should there be an AND or OR?

Reply:
just to clarify 
![image](https://user-images.githubusercontent.com/74790101/158625874-167e3842-33b3-49f1-af98-895bc695a7de.png)
This doesnt need to be **this** because it is not in a culture block right?


Also, what do I do here? Should I put all **culture_group** things in **culture** as well?
![image](https://user-images.githubusercontent.com/74790101/158626121-3e38969d-787c-42ef-bd57-7c1f24e7ab3b.png)

Reply:
> should there be an AND or OR?

Neither. There's no need for AND because by default all the triggers must be true.

> This doesnt need to be this because it is not in a culture block right?

It doesn't. Correct, it is not in culture block.

> Also, what do I do here? Should I put all culture_group things in culture as well?

No. These triggers are nested in character scope. You can access both culture and culture group in character scope.